### DISCUSSION TITLE
G6 Transmitter Connection Problems

### DISCUSSION THREAD

**Discussion by Lynd** [Navid200/xDrip#69] [Dexcom]
I just changed my G6 sensor and transmitter (because transmitter was > 100 days and hard resets failed). Now, my transmitter will not connect, even after restarting the collector. [](url)  

I receive errors "authentication failed!!!" and "could not verify challenge". There are other errors that indicate "Disconnected from MAC=.... with status 19". I tried rebooting my phone with no success.  

Here are screenshots of my System Status pages:

![temp](https://user-images.githubusercontent.com/8272764/134776571-5d655916-d2c6-4eac-8a25-39d584711970.jpg)
![temp2](https://user-images.githubusercontent.com/8272764/134776572-dad42d02-a02c-42b2-85c5-f2859362931a.jpg)
s

**Comment by Navid200** [comment]
@Lynd You have entered B as the first character of your transmitter serial number.
That cannot be!
The serial number of a G5 starts with 4.

Can you show a screenshot of the transmitter box?

If it's a G6, the serial number starts with 8.

**Comment by Navid200** [comment]
@Lynd You say that it is a G6.  Then, you need to change the settings as shown here:
https://github.com/Navid200/xDrip/wiki/G6-Recommended-Settings

According to the classic status page you are showing, you have set it up as a G5.

**Reply by Lynd** [reply]
I had misread the "8" as "B" (－‸ლ) .   Fixed that, but it still shows as "G5".  A pop screen shows "Collecting Initial Recordings".  I reset the collector, but the Initial Recordings pop-up is still present. 
![temp](https://user-images.githubusercontent.com/8272764/134779606-5124a804-7387-4413-aac4-8b0f6813119d.jpg)
![temp2](https://user-images.githubusercontent.com/8272764/134779617-d9ac0bde-cd93-470e-ac05-55d461470128.jpg)

**Comment by Navid200** [comment]
@Lynd Can you show a screenshot of the following page?
Settings -> G5/G6 Debug Settings

**Reply by Lynd** [reply]
Sure. Here it is
![temp](https://user-images.githubusercontent.com/8272764/134779777-cafd9da3-c856-4c22-990f-096b13d57bed.jpg)

**Comment by Navid200** [comment]
Those settings are incorrect.
These are the settings again:
https://github.com/Navid200/xDrip/wiki/G6-Recommended-Settings

You should be able to correctly set them.  If there is a reason you cannot, please tell me why.

**Comment by Lynd** [comment]
Not sure how I missed that. Sorry.  Now, the pop-up is gone and main screen shows "stopped" and "sensor session does not appear to be started".

**Reply by Navid200** [reply]
If all settings are correct, and you see stopped as the status, you have 2 options.

1.  You can start sensor.  If you are lucky, you will have readings after the 2-hour warm-up.
If you don't have readings then, you will need to perform a restart:
https://github.com/Navid200/xDrip/wiki/Restart-G6-Sensor

2.  You can go ahead and perform a restart now

**Comment by Lynd** [comment]
Thank you. I did a restart right away and started getting readings.   Those mistakes I made caused a really stressful 24 hours, but all is okay now.

**Reply by Navid200** [reply]
Great
Well done
